tools/knarc/Makefile

```diff
@@ -8,7 +8,7 @@ else
 C_SRCS   :=
 UNAME_S  := $(shell uname -s)
 ifeq ($(UNAME_S),Darwin)
-LDFLAGS  += -lstdc++ -lc++ -lc
+LDFLAGS  += -lstdc++ -lc++ -lc /usr/local/opt/llvm@8/lib/libc++fs.a -O2 -Wall -Wno-switch
```

Comment:
optimisation and warning flags for ld?

Reply:
This in particular links the static `libc++fs.a` library, to avoid issues due to missing symbols in 10.14:

```
Undefined symbols for architecture x86_64:
  "std::__1::__fs::filesystem::directory_iterator::__dereference() const", referenced from:
(...)
```

All this is because of using filesystem/experimental which is not a feature supported natively by macOS until 10.15.

(And the `-02 -Wall -Wno-switch` stuff is a typo as well, basically duplicated flags - Removed now)

If you were not to use LLVM's clang you would get a lots of warnings like these:

```
In file included from Source.cpp:5:
./Narc.h:81:25: error: 'path' is unavailable: introduced in macOS 10.15
    bool Pack(const fs::path& fileName, const fs::path& directory);
```